src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -225,6 +225,13 @@ public void should_parse_language_czech(string postTitle)
             result.Language.Id.Should().Be(Language.Czech.Id);
         }
 
+        [TestCase("Castle.2009.S01E14.Ukrainian.HDTV.XviD-LOL")]
+        [TestCase("True.Detective.S01E01.1080p.WEB-DL.Urk.Eng.TVKlondike")]
```

Comment:
The testcase says `urk` and the regex matches on `ukr`...

Reply:
Oops. Have not been able to run test locally...